holidays/calendars/thai.py

```diff
@@ -307,32 +321,32 @@ def _get_start_date(self, year: int) -> Optional[date]:
         return _timedelta(_ThaiLunisolar.START_DATE, delta_days)
 
     def makha_bucha_date(self, year: int, calendar=None) -> Optional[date]:
-        """
-        Calculate the estimated Gregorian date of Makha Bucha.
-        If the Gregorian year input is invalid, this will outputs None instead.
+        """Calculate the estimated Gregorian date of Makha Bucha.
 
         Also known as "Magha Puja", "Makha Buxha" and "Meak Bochea".
         This coincides with the 15th Waxing Day of Month 3
         in Thai Lunar Calendar, or Month 4 in Athikamat years.
 
         KHMER_CALENDAR will always use Month 3 regardless of year type.
 
-        To calculate, we use use the following time delta:
+        To calculate, we use the following time delta:
         - Athikamat: 15th Waxing Day of Month 4
                      or 29[1] + 30[2] + 29[3] + 15[4] -1 = 102
         - Athikawan: 15th Waxing Day of Month 3
                      or 29[1] + 30[2] + 15[3] -1 = 73
         - Pakatimat: 15th Waxing Day of Month 3
                      or 29[1] + 30[2] + 15[3] -1 = 73
```

Comment:
Such parts don't look pretty (just as a single paragraph), but I'm not sure how to make them better...